cmd/sign_test.go

```diff
@@ -28,8 +32,31 @@ var _ = Describe("SignCmd", func() {
 	)
 
 	It("with valid args", func() {
+		old := os.Stdout // keep backup of the real stdout
+		r, w, _ := os.Pipe()
+		os.Stdout = w
```

Comment:
It's probably easier to use `boshlog.NewWriterLogger` and pass a `bytes.NewBuffer`.

Reply:
@bgandon and I paired on this a bit, and this change was not feasible, mostly because it's not a logger problem, we actually want to capture output of a fmt.Printf [here](https://github.com/cloudfoundry/bosh-davcli/blob/master/cmd/sign.go#L38).